.github/workflows/subsurface.yml

```
@@ -73,10 +73,10 @@ jobs:
       env:
         # If you want the CI to (temporarily) run against your fork of the testdada,
         # change the value her from "equinor" to your username.
-        TESTDATA_REPO_OWNER: equinor
+        TESTDATA_REPO_OWNER: CeetronSolutions
         # If you want the CI to (temporarily) run against another branch than master,
         # change the value her from "master" to the relevant branch name.
-        TESTDATA_REPO_BRANCH: master
+        TESTDATA_REPO_BRANCH: EQ_293-VectorCalculatorConfig
```

Remember to change back before final merge
```suggestion
        TESTDATA_REPO_BRANCH: master
```